Service 'cvat_ui' failed build at a VM(Ubuntu 18.04 OS) on Google Cloud

When I `sudo docker-compose build ` at VM (Ubuntu 18.04 OS) on Google Cloud, I got this error `Service 'cvat_ui' failed to build: The command '/bin/sh -c npm install' returned a non-zero code: 1`. Could somebody help me figure out what I should do next? I have read our issues and tried several ways to solve it. But it still did not work. Here are the logs. 


Building: /usr/local/bin/node /tmp/cvat-canvas/node_modules/node-gyp/bin/node-gyp.js rebuild --verbose --libsass_ext= --libsass_cflags= --libsass_ldflags= --libsass_library=
gyp info it worked if it ends with ok
gyp verb cli [
gyp verb cli   '/usr/local/bin/node',
gyp verb cli   '/tmp/cvat-canvas/node_modules/node-gyp/bin/node-gyp.js',
gyp verb cli   'rebuild',
gyp verb cli   '--verbose',
gyp verb cli   '--libsass_ext=',
gyp verb cli   '--libsass_cflags=',
gyp verb cli   '--libsass_ldflags=',
gyp verb cli   '--libsass_library='
gyp verb cli ]
gyp info using node-gyp@3.8.0
gyp info using node@14.15.0 | linux | x64
gyp verb command rebuild []
gyp verb command clean []
gyp verb clean removing "build" directory
gyp verb command configure []
gyp verb check python checking for Python executable "python2" in the PATH
gyp verb `which` failed Error: not found: python2
gyp verb `which` failed     at getNotFoundError (/tmp/cvat-canvas/node_modules/which/which.js:13:12)
gyp verb `which` failed     at F (/tmp/cvat-canvas/node_modules/which/which.js:68:19)
gyp verb `which` failed     at E (/tmp/cvat-canvas/node_modules/which/which.js:80:29)
gyp verb `which` failed     at /tmp/cvat-canvas/node_modules/which/which.js:89:16
gyp verb `which` failed     at /tmp/cvat-canvas/node_modules/isexe/index.js:42:5
gyp verb `which` failed     at /tmp/cvat-canvas/node_modules/isexe/mode.js:8:5
gyp verb `which` failed     at FSReqCallback.oncomplete (fs.js:183:21)
gyp verb `which` failed  python2 Error: not found: python2
gyp verb `which` failed     at getNotFoundError (/tmp/cvat-canvas/node_modules/which/which.js:13:12)
gyp verb `which` failed     at F (/tmp/cvat-canvas/node_modules/which/which.js:68:19)
gyp verb `which` failed     at E (/tmp/cvat-canvas/node_modules/which/which.js:80:29)
gyp verb `which` failed     at /tmp/cvat-canvas/node_modules/which/which.js:89:16
gyp verb `which` failed     at /tmp/cvat-canvas/node_modules/isexe/index.js:42:5
gyp verb `which` failed     at /tmp/cvat-canvas/node_modules/isexe/mode.js:8:5
gyp verb `which` failed     at FSReqCallback.oncomplete (fs.js:183:21) {
gyp verb `which` failed   code: 'ENOENT'
gyp verb `which` failed }
gyp verb check python checking for Python executable "python" in the PATH
gyp verb `which` failed Error: not found: python
gyp verb `which` failed     at getNotFoundError (/tmp/cvat-canvas/node_modules/which/which.js:13:12)
gyp verb `which` failed     at F (/tmp/cvat-canvas/node_modules/which/which.js:68:19)
gyp verb `which` failed     at E (/tmp/cvat-canvas/node_modules/which/which.js:80:29)
gyp verb `which` failed     at /tmp/cvat-canvas/node_modules/which/which.js:89:16
gyp verb `which` failed     at /tmp/cvat-canvas/node_modules/isexe/index.js:42:5
gyp verb `which` failed     at /tmp/cvat-canvas/node_modules/isexe/mode.js:8:5
gyp verb `which` failed     at FSReqCallback.oncomplete (fs.js:183:21)
gyp verb `which` failed  python Error: not found: python
gyp verb `which` failed     at getNotFoundError (/tmp/cvat-canvas/node_modules/which/which.js:13:12)
gyp verb `which` failed     at F (/tmp/cvat-canvas/node_modules/which/which.js:68:19)
gyp verb `which` failed     at E (/tmp/cvat-canvas/node_modules/which/which.js:80:29)
gyp verb `which` failed     at /tmp/cvat-canvas/node_modules/which/which.js:89:16
gyp verb `which` failed     at /tmp/cvat-canvas/node_modules/isexe/index.js:42:5
gyp verb `which` failed     at /tmp/cvat-canvas/node_modules/isexe/mode.js:8:5
gyp verb `which` failed     at FSReqCallback.oncomplete (fs.js:183:21) {
gyp verb `which` failed   code: 'ENOENT'
gyp verb `which` failed }
gyp ERR! configure error 
gyp ERR! stack Error: Can't find Python executable "python", you can set the PYTHON env variable.
gyp ERR! stack     at PythonFinder.failNoPython (/tmp/cvat-canvas/node_modules/node-gyp/lib/configure.js:484:19)
gyp ERR! stack     at PythonFinder.<anonymous> (/tmp/cvat-canvas/node_modules/node-gyp/lib/configure.js:406:16)
gyp ERR! stack     at F (/tmp/cvat-canvas/node_modules/which/which.js:68:16)
gyp ERR! stack     at E (/tmp/cvat-canvas/node_modules/which/which.js:80:29)
gyp ERR! stack     at /tmp/cvat-canvas/node_modules/which/which.js:89:16
gyp ERR! stack     at /tmp/cvat-canvas/node_modules/isexe/index.js:42:5
gyp ERR! stack     at /tmp/cvat-canvas/node_modules/isexe/mode.js:8:5
gyp ERR! stack     at FSReqCallback.oncomplete (fs.js:183:21)
gyp ERR! System Linux 5.4.0-1028-gcp
gyp ERR! command "/usr/local/bin/node" "/tmp/cvat-canvas/node_modules/node-gyp/bin/node-gyp.js" "rebuild" "--verbose" "--libsass_ext=" "--libsass_cflags=" "--libsass_ldflags=" "--libsass_library="
gyp ERR! cwd /tmp/cvat-canvas/node_modules/node-sass
gyp ERR! node -v v14.15.0
gyp ERR! node-gyp -v v3.8.0
gyp ERR! not ok 
Build failed with error code: 1
npm info lifecycle node-sass@4.13.1~postinstall: Failed to exec postinstall script
npm timing action:postinstall Completed in 834ms
npm timing stage:rollbackFailedOptional Completed in 199ms
npm timing stage:runTopLevelLifecycles Completed in 21429ms
npm WARN cvat-canvas@2.1.3 No repository field.
npm WARN cvat-canvas@2.1.3 scripts['server'] should probably be scripts['start'].
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@2.1.3 (node_modules/watchpack/node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@2.1.3: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@1.2.9 (node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.2.9: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! node-sass@4.13.1 postinstall: `node scripts/build.js`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the node-sass@4.13.1 postinstall script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
npm timing npm Completed in 22973ms

npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2020-10-28T05_56_36_720Z-debug.log

--- a/cvat-canvas/package.json
+++ b/cvat-canvas/package.json
@@ -33,7 +33,7 @@
     "eslint-config-airbnb-typescript": "^4.0.1",
     "eslint-config-typescript-recommended": "^1.4.17",
     "eslint-plugin-import": "^2.18.2",
-    "node-sass": "^4.13.1",
+    "node-sass": "^4.14.1",
     "nodemon": "^1.19.4",
     "postcss-loader": "^3.0.0",
     "postcss-preset-env": "^6.7.0",

--- a/cvat-ui/package.json
+++ b/cvat-ui/package.json
@@ -33,7 +33,7 @@
     "eslint-plugin-react": "^7.21.5",
     "eslint-plugin-react-hooks": "^4.2.0",
     "html-webpack-plugin": "^3.2.0",
-    "node-sass": "^4.13.0",
+    "node-sass": "^4.14.1",
     "postcss-loader": "^3.0.0",
     "postcss-preset-env": "^6.7.0",
     "react-svg-loader": "^3.0.3",

